DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/Image.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import CoreGraphics
+import SwiftUI
+
+/// Represents a SwiftUI.GraphicsImage
+@available(iOS 13.0, tvOS 13.0, *)
+internal struct GraphicsImage {
+    let contents: Contents
+    let scale: CGFloat
+    let orientation: SwiftUI.Image.Orientation
+
+    enum Contents {
+        case cgImage(CGImage)
+        case unknown
+    }
+}
+
+@available(iOS 13.0, tvOS 13.0, *)
+extension GraphicsImage.Contents: Equatable {
```

Comment:
/comment I would refrain from adding such compliance only for tests. If there is no need in business logic, lets move it to the test target.